integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/api/MessageProcessorWSATestAPI.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+ ~  Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ ~
+ ~  WSO2 Inc. licenses this file to you under the Apache License,
+ ~  Version 2.0 (the "License"); you may not use this file except
+ ~  in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing,
+ ~  software distributed under the License is distributed on an
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+ ~
+ -->
+
+<api xmlns="http://ws.apache.org/ns/synapse" name="MessageProcessorWSATestAPI" context="/test">
```

Comment:
Let's not have the generic context name "test", or else this could cause conflicts with other test cases.

Reply:
done